src/App.vue

```diff
@@ -539,6 +544,18 @@ const startResize = (event) => {
   document.body.style.userSelect = "none"
 }
 
+const { vueFlowRef } = useVueFlow();
+const { capture } = useScreenshot();
+
+function doPNGscreenshot() {
```

Comment:
If we are going to try for any sort of consistency this function should be named `doPngScreenshot`.